docs/scaffold-a-new-project/README.md

```diff
@@ -0,0 +1,72 @@
+# Scaffold a New Project
+
+Neutrino provides a scaffolding tool to help you quickly jump-start a project. `create-neutrino-project` uses
+presets behind the scene to build projects. If you are not familiarized with them, take a moment to
+explore [what presets are](https://neutrino.js.org/presets/).
+
+## Getting Started
+Run the following command to start the interactive session. Substitute `<directory-name>` with the directory name you wish to create for this project.
```

Comment:
Session feels like the wrong word here, but I'm not sure what the right word is. Open to suggestions.

Reply:
s/interactive session/process/?